### DISCUSSION TITLE
请问，如何更改进度条缩略图的大小？

### DISCUSSION THREAD

**Discussion by philip-fhjkladsh** [hooke007/mpv_PlayKit#527] [Q&A 问答] [answered]
目前的缩略图感觉有点大，我想稍微改小一点。或者怎么关掉？

**Comment by hooke007** [comment, marked as answer]
去看脚本选项

**Reply by philip-fhjkladsh** [reply]
谢谢。光在mpvconf里找了。第一次用，都是现学的。

**Comment by ligs85** [comment]
我也觉得默认太大了，发现在script-opts.conf可以调像素大小。
但其实我更想关掉... 没有发现开关设置...

**Reply by hooke007** [reply]
脚本选项里写了

**Reply by ligs85** [reply]
谢谢，找到了
script-opts-append = thumbfast-auto_run=no
